src/challenges.js

```diff
@@ -2,17 +2,26 @@
 const each = (elements, cb) => {
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
+  return elements.forEach((units, i) => cb(units, i));
```

Comment:
Sorry the instructions I gave out weren't clear. You were supposed to not use forEach to create each and map to create map..